solidity/contracts/converter/LiquidityPoolConverter.sol

```diff
@@ -0,0 +1,74 @@
+pragma solidity 0.4.26;
+import './BancorConverter.sol';
+
+/**
+  * @dev Liquidity Pool Converter
+  * 
+  * The liquidity pool converter is the parent contract for specific types of converters that
+  * manage liquidity pools.
+  *
+  * Liquidity pools have 2 reserves or more and they allow converting between them.
+*/
+contract LiquidityPoolConverter is BancorConverter {

```

Comment:
@yudilevi : Don't we want to declare abstract functions `addLiquidity` and `removeLiquidity` here?

Reply:
Second thought, that would force us to find a generalization of these two functions for both V1 and V2, which is exactly what we've been trying to avoid.
So I guess we should probably avoid this, right?